src/index.tsx

```diff
@@ -14,8 +14,9 @@ export type MediapipeCameraProps = {
 export const MediapipeCamera: React.FC<MediapipeCameraProps> = ({
   style,
   processor,
+  activeCamera,
```

Comment:
This doesn't build, because you need to add activeCamera as a prop. I am extremely surprised that you did not see red squiggles in vscode. I think that we need to discuss online because you should not have been able to submit this PR, the commit process should have caused a lint failure.